NEWS.md

```diff
@@ -1,3 +1,7 @@
+## 0.2.4
+
+* Support for the [`cubbyhole`](https://www.vaultproject.io/docs/secrets/cubbyhole/index.html) secret engine and respone wrapping
+
```

Comment:
Fix typo on "respone" :-)
And if you like, add "Fix connection timout on windows" because it is an amazing fix...

Reply:
This is updated now